docs/Catalog/VisualizationDashboards.md

```diff
@@ -285,7 +286,52 @@ Load More to fetch up to about 6MB of zipped data. Beyond this size, click Downl
 to see the entire file contents.
 
 In order to open the analysis and visualization capabilities of Perspecitve,
-click the vertical ellipsis, upper left. Click, drag, and drop columns to explore
-the data.
+click the vertical ellipsis, upper left. You can hold this settings control opened
+on page load in files showed with `quilt_summarize.json` by providing additional
+configuration:
+
+```json
+// quilt_summarize.json
+[
+  {
+    "path": "file1.csv",
+    "types": [
+      {
+        "name": "perspective",
+        "config": {
+          "settings": true
+        }
+      }
+    ]
+  }
+]
+```
+
+Besides `{ "settings": true }` you can provide any object previously saved from Perspective.
+
+![](../imgs/perspective-save.png)
+
+All filters and columns will be restored:
+
+```json
+// quilt_summarize.json
+[
+  {
+    "path": "file1.csv",
+    "types": [
+      {
+        "name": "perspective",
+        "config": {
```

Comment:
Is .config *exactly* what gets passed to `.restore`? If so this is OK, otherwise we probably need to introduce a `config.restore` property here? Basically I want ot make sure the `settings` key is expected by perspective in a `restore` call.

Reply:
Yes, `config` value from `quilt_summarize.json` gets passed to `.restore()` method without changes.

`settings` is part of config, there is even note in documentation for `toggleConfig` method:
> This method is equivalent to viewer.restore({settings: force})